mtp_api/apps/core/tests/test_upload_dump_for_ap.py

```diff
@@ -0,0 +1,29 @@
+import boto3
+
+from moto import mock_s3
+from django.core.management import CommandError, call_command
+from django.test import TestCase
+
+
+class UploadDumpForAp(TestCase):
+    def test_invalid_arguments(self):
+        with self.assertRaises(CommandError, msg='Should not be missing path to file'):
+            call_command('upload_dump_for_ap')
+
+        with self.assertRaises(CommandError, msg='Should not be missing bucket name'):
+            call_command('upload_dump_for_ap', 'mtp_api/apps/core/fixtures/test_file.txt')
+
+    @mock_s3
```

Comment:
i think `moto` isn't particularly necessary here and the test could be a lot simpler. in essence, the uploading command just passes string from the command line directly to the boto library without processing. mocking in this way involves pretending to create a bucket, upload a file and then downloading it: 2 of those steps do not happen in the management command and the third is entirely handled by boto. what i think would be better is adding a couple of lines to the other test such that `upload_to_s3` is mocked and the check is simply to see that it was called with the expected generated arguments